src/core/ScaleManager.js

```diff
@@ -725,7 +725,7 @@ Phaser.ScaleManager.prototype = {
 
         this._booted = true;
 
-        if (this._pendingScaleMode)
+        if (this._pendingScaleMode !== undefined)
```

Comment:
Should consider instead `!= undefined` or `!= null`, so it also catches `null` too.